metricbeat/module/elasticsearch/shard/shard.go

```diff
@@ -31,7 +31,7 @@ func init() {
 }
 
 const (
-	statePath = "/_cluster/state/version,nodes,master_node,routing_table"
+	statePath = "/_cluster/state/version,nodes,master_node,routing_table?local=true"
```

Comment:
Would we miss data in a `scope: cluster` scenario where we only have one metricbeat process ? i'd expect `local=true` to be set only when `scope: node`

Reply:
While developing this, I compared the results of the API with and without this flag and they were the same.
@DaveCTurner Can you confirm if that's always the case?

Reply:
No (or at least this change doesn't make a meaningful difference in that regard). The `?local=true` parameter means "give me your local copy of the entire cluster state rather than forwarding this request to the elected master" but you get the same state either way (in fact sometimes you'll get a slightly fresher state with `?local=true`, and sometimes `?local=true` will succeed when `?local=false` would fail, so this seems like the right change)

Reply:
"No" as in:

> Would we miss data in a `scope: cluster` scenario where we only have one metricbeat process ?

No

> Can you confirm if that's always the case?

Yes

Reply:
Thanks for the insight!